feature-libs/product-configurator/rulebased/styles/_configurator-attribute-header.scss

```diff
@@ -51,4 +51,18 @@
     padding-block-start: 5px;
     display: block;
   }
+
+  .cx-header-label-row {
+    display: flex;
+    @include media-breakpoint-down(xs) {
```

Comment:
Please, remove this restriction because **_Show Options_** button should always be displayed underneath the attribute name.